web/index.html

```diff
@@ -31,26 +72,23 @@
     <div class="container-fluid no-horiz-padding">
         <pre class="log"></pre>
     </div>
-
-
-    <script src="__PATH__/socket.io/socket.io.js"></script>
-    <script src="__PATH__/tinycon.min.js"></script>
-    <script src="__PATH__/ansi_up.js"></script>
-    <script src="__PATH__/app.js"></script>
-    <script type="text/javascript">
-        var socket = new io.connect('/__NAMESPACE__', {
-            path: '__PATH__/socket.io',
-            transports: ['websocket']
-        });
-
-        window.load = App.init({
-            socket: socket,
-            container: document.getElementsByClassName('log')[0],
-            filterInput: document.getElementsByClassName('query')[0],
-            pauseBtn: document.getElementsByClassName('btn-pause')[0],
-            topbar: document.getElementsByClassName('topbar')[0],
-            body: document.getElementsByTagName('body')[0]
-        });
+    <script>
+        function getProcess(){
+            var xmlHttp = new XMLHttpRequest();
+            xmlHttp.open( "GET", 'http://127.0.0.1:8000/api/process/list', false );
```

Comment:
Again the URL needs to be relative if it is assumed that the portal will be hosted on the same domain. 
Otherwise use the actual domain.